src/witan/models/dem/ccm/fert/fertility.clj

```diff
@@ -138,6 +183,26 @@
                           :fert-rate :births)]
     {:births-by-age-sex-mother projected-births}))
 
+(defworkflowfn project-births-1-1-0
```

Comment:
yes, it's because proj-asfr-1-0-0 had a different output schema than proj-asfr-1-1-0. If we axe the old profj-asfr then we can get rid of this one too.